cleanrl/ppo2_continuous_action.py

```diff
@@ -208,8 +216,19 @@ def __init__(self):
 
     def forward(self, x):
         x = preprocess_obs_fn(x)
-        x = torch.tanh(self.fc1(x))
-        x = torch.tanh(self.fc2(x))
+
+        x = self.fc1(x)
+        if args.pol_layer_norm:
+            x = self.ln1(x)
+        x = torch.tanh(x)
+
+        x = self.fc2(x)
+
+        if args.pol_layer_norm:
+            x = self.ln2(x)
+
+        x = torch.tanh(x)
+
```

Comment:
Delete the spaces, please. They seem fairly random.

Reply:
My bad. I was probably "high" when working on that part hahaha